homeassistant/components/overkiz/config_flow.py

```diff
@@ -144,9 +140,15 @@ async def async_step_zeroconf(
         gateway_id = properties["gateway_pin"]
 
         LOGGER.debug("ZeroConf discovery detected gateway %s", obfuscate_id(gateway_id))
+        return await self._process_discovery(gateway_id)
 
+    async def _process_discovery(self, gateway_id: str) -> FlowResult:
+        """Handle discovery of a gateway."""
         await self.async_set_unique_id(gateway_id)
         self._abort_if_unique_id_configured()
+        self.context["title_placeholders"] = {
+            CONF_NAME: f"Gateway: {obfuscate_id(gateway_id)}"
+        }
```

Comment:
Could you add a test to see if this placeholder is present?

Reply:
Code coverage is still 100%, so apparently this is not necessary.

Reply:
It would still be good to add a test